### ISSUE TITLE
Generating ms-output from tree sequence

### ISSUE THREAD

**Issue by SBNoor** [tskit-dev/msprime#762]
I have tree sequence recording from SLiM. Is there a way in msprime to generate the MS output directly from msprime?

**Comment by jeromekelleher** [comment]
This isn't something that we support in msprime, but there is a new [tsconvert](https://github.com/tskit-dev/tsconvert) repo in development which is intended to do all of these types of conversions between formats. Unfortunately tsconvert hasn't been released and is pretty incomplete. There is a [ts_ms](
https://github.com/tskit-dev/tsconvert/blob/master/tsconvert/newick.py#L105) function which might be useful to you though.

**Comment by bhaller** [comment]
Hi @jeromekelleher.  Regarding the answer above, I see the last commit to tsconvert was 17 months ago; is it still considered an active project or has it been abandoned?  I just got another SLiM user today who wants to get MS output from a tree sequence; this comes up with some regularity.  This would really not be hard to add to the Python APIs, would it?  It's a very simple format; it'd be nice to have a better solution for this.

**Comment by jeromekelleher** [comment]
Yeah, this is a pretty simple thing to do and we should just do it in tskit. I guess we can just thrown an error if we hit a situation where ms-format doesn't make sense.

We haven't abandoned tsconvert, as such, just nobody has done any work on it! ;)
